src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -36,6 +35,10 @@ public void getStatistic(String fromFileName, String toFileName) {
     }
 
     private void writeToFile(String toFileName, int supply, int buy) {
+        report(toFileName, supply, buy);
+    }
+
+    private void report(String toFileName, int supply, int buy) {
```

Comment:
```suggestion
    private void createReport(int supply, int buy) {
```
this method should only create report and not write to file, you have another method for this